LoopKitUI/Views/CorrectionRangeScheduleEditor.swift

```diff
@@ -65,14 +90,22 @@ public struct CorrectionRangeScheduleEditor: View {
                 )
             },
             actionAreaContent: {
+                instructionalContentIfNecessary
                 guardrailWarningIfNecessary
             },
             savingMechanism: .synchronous { items in
                 let quantitySchedule = DailyQuantitySchedule(unit: self.unit, dailyItems: items)!
                 let rangeSchedule = GlucoseRangeSchedule(rangeSchedule: quantitySchedule, override: self.initialSchedule?.override)
                 self.save(rangeSchedule)
-            }
+            },
+            mode: mode
         )
+        .onTapGesture {
```

Comment:
This was the best way I could find to update the `userHasEdited` binding variable; I'd love to update it more frequently if possible, but I couldn't find a way to do that in SwiftUI

Reply:
The above comment about using a comparison instead of a tracking variable should resolve this.